python/pyspark/mllib/stat/_statistics.py

```diff
@@ -33,34 +39,34 @@ class MultivariateStatisticalSummary(JavaModelWrapper):
     Trait for multivariate statistical summary of a data matrix.
     """
 
-    def mean(self):
-        return self.call("mean").toArray()
+    def mean(self) -> ndarray:
+        return cast(JavaObject, self.call("mean")).toArray()
```

Comment:
a lot of `cast(JavaObject, self.call("..."))` added because `bytearray` and `bytes` does not have `toArray`